src/Core/Checkout/Cart/Event/CartCreatedEvent.php

```diff
@@ -12,13 +13,24 @@ class CartCreatedEvent extends Event
      */
     protected $cart;
 
-    public function __construct(Cart $cart)
+    /**
+     * @var SalesChannelContext
+     */
+    protected $context;
+
+    public function __construct(Cart $cart, SalesChannelContext $context)
     {
         $this->cart = $cart;
+        $this->context = $context;
     }
 
     public function getCart(): Cart
     {
         return $this->cart;
     }
+
+    public function getContext(): SalesChannelContext
```

Comment:
would be nice if you could implement `Shopware\Core\Framework\Event\ShopwareEvent` and make two methods. one which returns the SalesChannelContext and one which returns the Context

Reply:
`Shopware\Core\Framework\Event\ShopwareEvent` is implemented